docs/source/visualization/pharos.rst

```diff
@@ -0,0 +1,48 @@
+#####################
+ Pharos Installation
+#####################
+
+To install Pharos SDK by **pip**, run the following command:
+
+.. code:: console
+
+   $ pip3 install pharos
+
+
+##############
+ Pharos Usage
+##############
+
+Load a Open Dataset Instance
```

Comment:
There is no need to separate datasets into open datasets and general datasets. 
Merge them into one section.